src/commons/components/page-skeleton/PageSkeletonOrgTeam.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div class="flex">
+    <section class="min-h-full bg-blue-11 relative"><OrgNavbarTeam /></section>
+    <section class="flex flex-col w-full h-full">
+      <PageNavbar />

```

Comment:
Why don't use PageSkeleton here instead of PageNavbar and PageFooter

Reply:
Because we will need to change PageNavbar to one for organization instead.